18F/fec-cms
# Misc updates

* Update install instructions
* Remove dependency on bower
* Simplify HTML tag whitelister

To verify:
* Check that app runs without bower
* Check that arbitrary HTML tags and attributes are allowed in the HTML editor

cc @noahmanger
